lib/nats.js

```diff
@@ -588,7 +588,19 @@ Client.prototype.connectCB = function() {
     this.flushPending();
 };
 
+/**
+ * @api private
+ */
+Client.prototype.cancelHeartbeat = function() {
+    if (this.pingTimer) {
```

Comment:
Should we not set pingTimer to null or undefined in case this gets called again?